hibernate-core/src/main/java/org/hibernate/annotations/AnyMetaDef.java

```diff
@@ -37,7 +37,7 @@
 	 * Names the discriminator Hibernate Type for this Any/ManyToAny mapping.  The default is to use
 	 * {@link org.hibernate.type.StringType}
 	 */
-	String metaType();
+	String metaType() default "string";
```

Comment:
@johnlinp, please add a test case that exercises this default.

Reply:
OK, I'll do it.

Reply:
I removed the property `metaType = "string"` in the existing unit tests to exercise my change.